docs/LogGuide.md

```diff
@@ -1062,40 +1728,94 @@ Examples:
+```
+
+#### Examples
 
 ```log
-26:12345678:PlayerOne:3C503C1C:24136:24136:9045:10000:4:0:-0.4730835:-158.1598:-23.9:3.110625:03E8:45:0:020130:0:106501CA:0129:4172D113:106501CA:012A:4168C8B4:106501CA:012B:40919168:106501CA:0232:40E00000:E0000000:
+Network Log Line Examples:
+38|2021-04-26T14:13:16.2760000-04:00|10FF0001|Tini Poutini|46504615|75407|75407|10000|10000|24|0|-645.238|-802.7854|8|1.091302|1500|3C|0|0A016D|41F00000|E0000000|1E016C|41F00000|E0000000||c1b3e1d63f03a265ffa85f1517c1501e
+38|2021-04-26T14:13:16.2760000-04:00|10FF0001||46504621|49890|49890|10000|10000|24|0|||||1500|3C|0||f62dbda5c947fa4c11b63c90c6ee4cd9
+38|2021-04-26T14:13:44.5020000-04:00|10FF0002|Potato Chippy|46504621|52418|52418|10000|10000|32|0|99.93127|113.8475|-1.862645E-09|3.141593|200F|20|0|0A016D|41F00000|E0000000|1E016C|41F00000|E0000000|0345|41E8D4FC|10FF0001|0347|80000000|10FF0002||d57fd29c6c4856c091557968667da39d
+
+ACT Log Line Examples:
+[14:13:16.276] 26:38:2021-04-26T14:13:16.2760000-04:00:10FF0001:Tini Poutini:46504615:75407:75407:10000:10000:24:0:-645.238:-802.7854:8:1.091302:1500:3C:0:0A016D:41F00000:E0000000:1E016C:41F00000:E0000000::c1b3e1d63f03a265ffa85f1517c1501e
+[14:13:16.276] 26:38:2021-04-26T14:13:16.2760000-04:00:10FF0001::46504621:49890:49890:10000:10000:24:0:::::1500:3C:0::f62dbda5c947fa4c11b63c90c6ee4cd9
+[14:13:44.502] 26:38:2021-04-26T14:13:44.5020000-04:00:10FF0002:Potato Chippy:46504621:52418:52418:10000:10000:32:0:99.93127:113.8475:-1.862645E-09:3.141593:200F:20:0:0A016D:41F00000:E0000000:1E016C:41F00000:E0000000:0345:41E8D4FC:10FF0001:0347:80000000:10FF0002::d57fd29c6c4856c091557968667da39d
 ```
 
+<!-- AUTO-GENERATED-CONTENT:END -->
+
 It seems likely that this line was added in order to extend functionality
 for the `18`, `1A`, and `1E` log lines without breaking previous content or plugins.
 
-## 27: NetworkUpdateHP
+### 27: NetworkUpdateHP
 
 It's not completely clear what triggers this log line,
 but it contains basic information comparable to `25` and `26`.
 It applies to allies and fairies/pets.
 
-Structure:
-`27:[Target ID]:[Target Name]:[Current HP]:[Max HP]:[Current MP]:[Max MP]:[Current TP]:[Max TP]:[position X]:[position Y]:[position Z]:[Facing]`
+<!-- AUTO-GENERATED-CONTENT:START (logLines:type=networkUpdateHP&lang=en) -->
 
-Examples:
+#### Structure
 
 ```log
-27:12345678:Eos:22851:22851:10000:10000:0:0:12.13086:-169.9398:-23.90031:-2.310888:
+Network Log Line Structure:
+39|2021-04-26T14:11:35.0000000-04:00|[id]|[name]|[currentHp]|[maxHp]|[currentMp]|[maxMp]|||[x]|[y]|[z]|[heading]
+
+ACT Log Line Structure:
+[14:11:35.0] 27:39:2021-04-26T14:11:35.0000000-04:00:[id]:[name]:[currentHp]:[maxHp]:[currentMp]:[maxMp]:::[x]:[y]:[z]:[heading]
 ```
 
+#### Examples
+
+```log
+Network Log Line Examples:
+39|2021-04-26T14:12:29.5060000-04:00|10FF0001|Tini Poutini|121087|121087|10000|10000|0|0|-648.3234|-804.5252|8.570148|1.010669||a0cca3b79b0fd65541e3fa6b11fa386e
```

Comment:
Probably 2-3 lines here would be ok too.

Reply:
I included the first four lines here to demonstrate that in some cases these come in at 3 second intervals. Maybe it's better explained as plain text?

Reply:
Oh, I definitely missed what that was trying to demonstrate.  Plain text with fewer examples sounds better to me.